constraints.txt

```
@@ -5,3 +5,6 @@ PyYAML>=6.0.1,<6.1
 # to avoid version conflicts
 requests==2.25.1
 requests-retry-session>=0.1,<0.2
+kubernetes==26.1.0
+requests-oauthlib==0.8.0
+urllib3==1.26.20
```

If you are going to specify this here, we want it to match the version installed on the CSM 1.6.0 images.

```suggestion
urllib3==1.25.10
```